src/components/qa/subcomponents/UploadMediaCard.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import clothingImg from '../../../../dist/clothing.jpg';
```

Comment:
Unused import. It also doesn't exist in your pull request, so it causes a fatal error.